alicloud/resource_alicloud_ess_scalinggroup.go

```diff
@@ -77,6 +77,12 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
 				ForceNew: true,
 				MinItems: 0,
 			},
+			"multi_az_policy": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "PRIORITY",
+				ValidateFunc: validateAllowedStringValue([]string{"PRIORITY", "BALANCE"}),
+			},
```

Comment:
HI @ebabani Thanks for your contribution. Can you define two const parameters in the file `alicloud/extension_ess.go` to support these two optional values.

In addition, if this parameter doesn't support update, please set 'ForceNew' to it.